pkgs/development/python-modules/picos/default.nix

```diff
@@ -0,0 +1,32 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitLab
+, numpy
+, cvxopt
+}:
+
+buildPythonPackage rec {
+  pname = "picos";
+  version = "1.2.0";
+
+  src = fetchFromGitLab {
+    owner = "picos-api";
+    repo = "picos";
+    rev = "b9c3105b31166c307296dee6e0c26cf6a0ab8723";
+    sha256 = "018xhc7cb2crkk27lhl63c7h77w5wa37fg41i7nqr4xclr43cs9z";
+  };
+
+  propagatedBuildInputs = [
+    numpy
+    cvxopt
+  ];
+
+  doCheck = false;
```

Comment:
Why are the tests disabled? Please include a comment.

Reply:
Sorry for my time to answer, I was not really available before. So there is no deep reason I guess, as I just got inspired by manual for that. But when I set `doCheck` to `true`... It fails. The errors are not fundamental, I just thing nix can't find the tests: 

```
running build_ext
usage: nix_run_setup [-h] [-u | -p] [-v] [-d | -q] [-l]
                     [-s SOLVER [SOLVER ...]] [-t SET [SET ...]]
                     [-c CASE [CASE ...]] [-n TEST [TEST ...]] [-m LEVEL] [-b]
                     [-o SET [SET ...]] [-x] [-f NUM | -F] [--objplaces NUM]
                     [--varplaces NUM]
nix_run_setup: error: unrecognized arguments: test
test (unittest.loader.ModuleImportFailure) ... ERROR

======================================================================
ERROR: test (unittest.loader.ModuleImportFailure)
----------------------------------------------------------------------
ImportError: Failed to import test module: test

```

However, I updated the check phase to:

```
checkPhase = ''
  python ${src}/test.py
'';
```

and now it seems to work with both python2 and python3. I don't know if it's the usual way to make tests... but at least it works. Let me know if it's good or not!